packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7096,15 +7127,24 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     let floorRequestNeeded = false;
+    if (tracks.screenShare?.video || (this.isMultistream && tracks.screenShare?.audio)) {
+      this.shareInstanceId = uuid.v4();
```

Comment:
see my comment on the other PR, like with the 'client.share.initiated' metric, we will want to create a new `shareInstanceId` also only when floorRequestNeeded is true